src/main/java/com/Alvaeron/MySQLManager.java

```diff
@@ -134,7 +135,11 @@ public void initDatabase() {
         });
     }
 
-    public void createRoleplayPlayer(final Player p) {
+    public void createRoleplayPlayer(final OfflinePlayer p) {
+        if (p.getUniqueId() == null || p.getName() == null) {
```

Comment:
This is a valid way of ending the method, but it would look better if it were checking for not null.  `if p != null and q != null then createRoleplayPlayer(...)`